rpm/podman.spec

```diff
@@ -179,17 +179,18 @@ capabilities specified in user quadlets.
 It is a symlink to %{_bindir}/%{name} and execs into the `%{name}sh` container
 when `%{_bindir}/%{name}sh` is set as a login shell or set as os.Args[0].
 
+%ifarch x86_64 aarch64
```

Comment:
Would it be better to define this condition once then store it as variable?
```
%ifarch x86_64 aarch64
machine_arch=1
%endif
```
And then do
```
%if %{defined machine_arch}
...
```

Reply:
I will rework it in a better way and close this PR.

Reply:
yes, would be more readable with the additional macro defined. Thanks for this PR.